mmv1/third_party/terraform/services/workbench/resource_workbench_instance_test.go.tmpl

```diff
@@ -850,3 +851,41 @@ resource "google_workbench_instance" "instance" {
 }
 `, context)
 }
+
+// TestAccWorkbenchInstance_metadataValidation ensures protected metadata keys cannot be set.
+func TestAccWorkbenchInstance_metadataValidation(t *testing.T) {
```

Comment:
You'll need to skip tests in VCR due to https://github.com/hashicorp/terraform-provider-google/issues/15427

```suggestion
func TestAccWorkbenchInstance_metadataValidation(t *testing.T) {
	acctest.SkipIfVcr(t) // Test doesn't interact with API
```